docset/windows/addsadministration/Add-ADGroupMember.md

```diff
@@ -329,6 +329,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisablePermissiveModify
+Group membership updates use permissive modify by default. This suppresses an error when adding a member that is already member of the group.
```

Comment:
```suggestion
Group membership updates use permissive modify by default. This suppresses an error when adding a member that is already a member of the group.
```

Reply:
Please excuse me, this is one of my first and until now biggest changes I make in github directly. I made a few mistakes, and I have created new pull requests and the one where I seem to have gotten things right are 2228 and 2229.

So I think we should cancel this one. You can check the other PRs and edit review them. Thank you.

Herbert Mauerer

Reply:
@e0i the author is asking here to close this PR as he created others to replace it, and also, this one has conflict merges.